backend/src/main/java/com/team4/backend/controller/SupervisorController.java

```diff
@@ -52,9 +52,9 @@ public SupervisorController(SupervisorService supervisorService) {
         return supervisorService.getAllAssignedStudentsForCurrentSemester(supervisorId);
     }
 
-    @GetMapping("/{email}")
+    @GetMapping("/{getByEmail}")
```

Comment:
why did you change it to getByEmail? Email was fine.

Reply:
there was a todo saying it needed to be changed to follow the convention

Reply:
🤷‍♂️